README.md

```diff
@@ -1,13 +1,13 @@
 # Happy Thoughts
```

Comment:
Just have to start by complementing your readme, they are usually skipped over. But yours is good!